content/docs/reference/quality-guidelines.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Quality Guidelines
+aliases:
+  - /docs/developing/quality-guidelines
+---
+Keep these guidelines in mind when creating or reviewing a pull request (PR). If you're not sure about a certain guideline when creating a PR, you can submit the PR as a draft and mention that you need help with a specific aspect of your PR. Many developers and community members are willing and able to help your PR improve!
+
+All PRs:
+- should have sound reasoning as to why they should be merged. This should be specified under the "Reason for changes" header when writing the PR description. Even if it seems obvious, others may need the additional context. PRs with little to no reason to be merged are likely to be delayed, ignored, or even closed.
+- must support both the Chromium and Firefox browser engines. Avoid using new or experimental web technologies unless a fallback is implemented that preserves feature parity. View the [unsupported browser](https://scratchaddons.com/unsupported-browser/) page to determine the current minimum versions that must be supported.
+- must be tested thoroughly. We have a very large userbase, so it's especially important that all possible use cases are considered. Every PR should be tested on a Chromium-based browser (Chrome, Edge, Brave, or another) **and** Firefox. Any tests done should be added to the "Tests" section of the PR description. **Org members, this is required in order to give approval.**
```

Comment:
What if you don't have access to the other browser yourself? What should the PR author or reviewers do in that case?

Reply:
Perhaps testing on both browsers could be a requirement for approving a PR but not for opening one.

Reply:
What I meant on this point was what Maximouse said.

Reply:
Reviewers can still give good feedback if they don't have access to both browsers, but full approval should require a thorough evaluation, and that can't happen if one of our supported browsers hasn't been tested.

Reply:
alr so i'm never approving prs again
got it 👍🏻

Reply:
Another option would be to only require one approval to test both browsers. That's likely the test coverage that Firefox currently receives.

Reply:
i think the best solution would be to require tests on both browsers for merging, but not for reviews. The majority of PRs would not have any difference on different browsers.

Reply:
I'd rather have this as a recommendation, and not a strict requirement. 90% of the case it will work on both browsers, so it is not something of a priority. At least that's on me.

Reply:
Agreed on making it a recommendation.

Reply:
I think it can be helpful to make sure complex code changes and changes that can sometimes be incompatible with one browser work on both browsers, but for simpler changes, I think testing on Chrome is enough. For the complex PRs like big addons, someone can request testing on Firefox before merging.

Or testing changes on Firefox could be put on a to-do list before the updates are released.

Reply:
https://github.com/ScratchAddons/ScratchAddons/issues/6567

This is why we want to make sure Firefox gets proper testing.

Reply:
Wait, is it a Firefox issue?

Reply:
> [ScratchAddons/ScratchAddons#6567](https://github.com/ScratchAddons/ScratchAddons/issues/6567)
> 
> This is why we want to make sure Firefox gets proper testing.

That's not a Firefox specific bug.